### PR TITLE
Challenge/santiago todo list

### PR DESCRIPTION
**Pull request by SantiagoMC52**

### CODE REVIEW

File: Santiago/challenge/todo-list/src/actions/actionCreators.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const taskAdded = () => {
```
Comment (shd8) [inline Santiago/challenge/todo-list/src/actions/actionCreators.js:1]: Review actions creators as we discussed at our meeting.

An action creator should return an action that goes to the dispatcher.

**Review by shd8 — APPROVED** [review]
You have to do hard corrections easy to apply. Do the changes we discussed and I'll review your pull request another time.

**Review by gteixi — APPROVED** [review]

**Review by IrenePerezAlonso — APPROVED** [review]

File: Santiago/challenge/todo-list/src/actions/actionCreators.js (hunk 1 above)
Comment (SantiagoMC52) [reply Santiago/challenge/todo-list/src/actions/actionCreators.js:1]: I appreciate your feedback, I will take it into account.

**Review by cristian-bars — APPROVED** [review]
Bona feina Santi!

**Pull request merged**